scene/gui/item_list.cpp

```diff
@@ -2009,9 +2135,15 @@ void ItemList::_bind_methods() {
 }
 
 ItemList::ItemList() {
-	scroll_bar = memnew(VScrollBar);
-	add_child(scroll_bar, false, INTERNAL_MODE_FRONT);
-	scroll_bar->connect(SceneStringName(value_changed), callable_mp(this, &ItemList::_scroll_changed));
+	scroll_bar_v = memnew(VScrollBar);
+	scroll_bar_v->set_name("scroll_bar_v");
```

Comment:
What's the name for?
We generally don't name internal Controls.

Reply:
I was using the ScrollContainer as a reference and it has default names for the scrollbars, so I thought they should be added.
I have removed them.